src/VCR/Cassette.php

```diff
@@ -68,17 +68,45 @@ public function hasResponse(Request $request)
      */
     public function playback(Request $request)
     {
-        foreach ($this->storage as $recording) {
-            $storedRequest = Request::fromArray($recording['request']);
-            if ($storedRequest->matches($request, $this->getRequestMatchers())) {
-                return Response::fromArray($recording['response']);
+        if ($this->config->getPlaybackMode() === VCR::PLAYBACK_MODE_ALL_IN_ORDER) {
+            if ($this->storage->valid()) {
+                $recording = $this->storage->current();
+                $this->storage->next();
+
```

Comment:
Explain how this is working. It looks like normal playback mode iterates over each recording, but 'all in order' just plays the first one? I must be missing something.